crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1831,7 +1792,11 @@ impl SymbolicationActor {
 
                     // Stackwalk the minidump.
                     let cfi = load_cfi_for_processor(cfi_caches);
-                    let minidump = ByteView::from_slice(&minidump);
+                    // we cannot map an `io::Error` into `MinidumpNotFound` since there is no public
+                    // constructor on `ProcessResult`. Passing in an empty buffer should result in
+                    // the same error though.
+                    let minidump =
+                        ByteView::open(minidump_path).unwrap_or_else(|_| ByteView::from_slice(b""));
```

Comment:
An empty buffer will now result in a generic "failed to read miniump" error with little information for debugging.

Is there no way you can wrap the error somehow?

Reply:
I think "minidump not found" is the appropriate error here. Also, not sure what to wrap this with, as also everything needs to go through serde, which `std::io::Error` does not.